packages/unlock-js/src/v6/getLock.js

```diff
@@ -79,6 +85,23 @@ export default async function (address, provider) {
   }
   delete update.tokenAddress
 
+  // check if compromised (Nov 22nd, 2021 attack)
+  if (update.unlockProtocol) {
```

Comment:
Let's move this logic to `unlock-app` please :)

Reply:
yep just kept the renaming